package.json

```diff
@@ -9,7 +9,7 @@
 		"node": ">=8.6"
 	},
 	"scripts": {
-		"test": "xo && nyc ava",
+		"test": "xo && nyc ava && tsc --noEmit --strict",
```

Comment:
I assumed `ts-node` already did this kind of validation? Aren't we running TS twice now?

https://github.com/sindresorhus/got/blob/985a3b2d384072bd7ddc9f2f89474cd3a41592a7/package.json#L103

Reply:
Yeah, but only one of them in strict mode. If we add `strict: true` to the `tsconfig.json` it will try to compile all the test code as strict mode as well. So this was the middle ground in the mean time, it does an extra compile of the library code in strict mode, just to check that it's conformant.

Reply:
It's not currently possible to have some parts non-strict: https://github.com/Microsoft/TypeScript/issues/28306

Reply:
Makes sense 👍